crates/matrix-sdk/Cargo.toml

```diff
@@ -113,6 +113,11 @@ git = "https://github.com/ruma/ruma"
 rev = "96155915f"
 features = ["client-api-c", "compat", "rand", "unstable-msc2448"]
 
+[dependencies.ruma-client-api]
+git = "https://github.com/ruma/ruma"
+rev = "96155915f"
+features = ["compat", "unstable-msc2965"]
```

Comment:
From what I could tell, this whole section is needed to add the msc2965 feature?

Reply:
Yeah, things that are behind an MSC are feature gated in Ruma land, not sure that you need the `compat` feature but I think the rust-sdk will enable that anyways.